### PR TITLE
change pycocotools to extras require

### PR DESCRIPTION
**Pull request by sainivedh**
Relax pandas, numpy, pyyaml, pillow, rich versions 
change pycocotools to extras_require (`pip install "clarifai[all]"`)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,6 +34,9 @@
     license="Apache 2.0",
     python_requires='>=3.8',
     install_requires=install_requires,
+    extras_require={
+        'data': ["pycocotools=2.0.6"],
```
Comment (sanjaychelliah) [inline setup.py:38]: can we have this as `pip install "clarifai[all]"` or  this is fine `pip install "clarifai[data]"` ?

Comment (sainivedh) [reply setup.py:38]: Both seems fine, since data-utilities are planned next, I thought of having `data`

Comment (sainivedh) [reply setup.py:38]: @sriram-clarifai your thoughts please

**Review by sanjaychelliah — APPROVED** [review]
LGTM

**Pull request merged**